doc/guides/dts/index.rst

```diff
@@ -346,44 +346,600 @@ Below is sample header content generated for this node, in
    #define DT_INST_0_NXP_KINETIS_SIM_LABEL             DT_NXP_KINETIS_SIM_40047000_LABEL
    #define DT_INST_0_NXP_KINETIS_SIM                   1
 
-Generated macro names follow the format ``DT_<node>_<property>``. For
-``DT_NXP_KINETIS_SIM_40047000_BASE_ADDRESS``, the node part is
-``NXP_KINETIS_SIM_40047000``, based on the compatible string that matched a
-binding for the node (``nxp,kinetis-sim``) and the node's unit address
-(``...@4004700``).
+All macros generated from devicetree start with ``DT_`` and use all-uppercase.
+Most macro names contain a node identifier, and macros that correspond to
+devicetree properties also contain an identifier for the property.
+
+Node identifiers in macro names
+-------------------------------
+
+Macros generated from particular devicetree nodes start with ``DT_<node>``,
+where ``<node>`` identifies the node. Several identifiers are generated for
```

Comment:
The introductory paragraph is written in terms of "identifiers" (without the `DT_`), but the body mostly uses "prefix"  (which has the `DT_`).

I think writing everything in terms of just one of them (my preference would be "identifier") would be more understandable.

Reply:
replaced prefix everywhere with identifier

Reply:
That's not quite what I'm saying. Only the `<node>` part is the identifier -- the part without the `DT_`. The entire `DT_<node>` is what's being called a "prefix".